Source/LightboxViewCell.swift

```diff
@@ -0,0 +1,21 @@
+import UIKit
+
+public class LightboxViewCell: UICollectionViewCell {
+
+  public static let reuseIdentifier: String = "LightboxViewCell"
```

Comment:
I agree with @vadymmarkov, adding it to the cell itself makes the view controller smaller if you add a lot of different cells. And using `reuseIdentifier` as the convention for storing it would make the process very linear. Heck you could even make a method that would return `Self` as a string so that you don't even have to add it yourself.

```swift
extension UICollectionViewCell {
  func reuseIdentifier() -> String {
    static func reuseIdentifier() -> String {
        return NSStringFromClass(self).componentsSeparatedByString(".").last!
    }
  }
}```

What do you guys think? @richardoti @RamonGilabert @vadymmarkov